Gemfile.lock

```diff
@@ -299,9 +299,6 @@ GEM
     standard (0.8.1)
       rubocop (= 1.0.0)
       rubocop-performance (= 1.8.1)
-    sweetalert2-rails (0.1.1)
-    sweetify (2.0.0)
-      actionpack
```

Comment:
I suspect that these were not checked in last time. I didn't adjust the Gemfile, but I'm happy to invesigate further if needed.